livelossplot/outputs/matplotlib_plot.py

```diff
@@ -92,13 +89,21 @@ def _default_before_plots(self, fig: plt.Figure, num_of_log_groups: int) -> None
         figsize_x = self.max_cols * self.cell_size[0]
         figsize_y = ((num_of_log_groups + 1) // self.max_cols + 1) * self.cell_size[1]
         fig.set_size_inches(figsize_x, figsize_y)
+        if num_of_log_groups < axes.size:
+            for idx, ax in enumerate(axes[-1]):
+                if idx >= (num_of_log_groups + len(self.extra_plots)) % self.max_cols:
+                    ax.set_visible(False)
```

Comment:
![image](https://user-images.githubusercontent.com/24765461/88277556-3c52f900-cce1-11ea-9c0a-fce2334d522d.png)
With this, not used charts are not visible. 
It should be a part of default_before_plots or sth other?

Reply:
Not sure if I understand the last comment. Is it something new?

Reply:
Currently, on master MatplotlibOutput raise an error in the case of an odd number of metrics (change above with `math.ceil`), because of a too-small number of rows. The purpose of this line is to delete charts from the last row which are not used. It is clear.

Reply:
Sure. In this case, did you fix that or intend to do so?

Reply:
Yes, I have fixed that